docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -818,6 +818,57 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[formatDateTime]
 include-tagged::{sql-specs}/docs/docs.csv-spec[formatTime]
 --------------------------------------------------
 
+
+[[sql-functions-datetime-to_char]]
+==== `TO_CHAR`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+TO_CHAR(
+    date_exp/datetime_exp/time_exp, <1>
+    string_exp) <2>
+--------------------------------------------------
+
+*Input*:
+
+<1> date/datetime/time expression
+<2> format pattern
+
+*Output*: string
+
+*Description*: Returns the date/datetime/time as a string using the format specified in the 2nd argument. The formatting
+pattern conforms to
+https://www.postgresql.org/docs/13/functions-formatting.html[PostgreSQL Template Patterns for Date/Time Formatting].
+If any of the two arguments is `null` or the pattern is an empty string `null` is returned.
+
+[NOTE]
+If the 1st argument is of type `time`, then the pattern specified by the 2nd argument cannot contain date related units
+(e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned.
+
+*Special Cases*
```

Comment:
Follows the way the `FORMAT` is written.

Reply:
This doesn't mean `FORMAT` has used the "proper" approach in this case, just like was the case for[ ordering the functions alphabetically](https://github.com/elastic/elasticsearch/pull/66486#discussion_r552081217). Using a `[NOTE]` instead of the current approach is not a hard rule afaik, but I would test converting those two statements in `[NOTE]`s, potentially look at other examples in the rest of the Elasticsearch documentation and decide if it's worth the change.

Reply:
Moved to `[NOTE]`